docs/troubleshooting-guide/timezones.md

```diff
@@ -8,60 +11,61 @@ The source of "wrong" numbers in charts or reports is often due to an underlying
+9. If by this point you have the correct value, it means your time zone was converted by the number of hours forward or backwards you manually set the filter. If not, then the problem might not be a direct time zone issue.
 
-For example, let's say have a PST server time zone, and a GMT reporting time zone. If you had to manually go back 9 hours to get correct numbers, that suggests that the conversion is not happening for some reason -- this suggests you are using timestamps without a time zone.
+Now that you have the time zone adjustment, go back to the earlier list of time zone questions and think about where this could have occurred. For example, let's say you have a PST server time zone and a GMT reporting time zone. If you had to manually go back 9 hours to get correct numbers, that suggests that the conversion is not happening for some reason, which in turn suggests you are using timestamps without a time zone.
 
-You can see a number of common problems below. If none of them apply, please [open a bug report](https://github.com/metabase/metabase/issues/new/choose) with the above information (time zones, and the results of the second troubleshooting process) as well as your Metabase, OS, and web browser versions.
+A few specific problems are described below. If none of them apply, please [open a bug report][metabase-file-bug] with the above information (time zones and the results of the second troubleshooting process) as well as your Metabase, OS, and web browser versions.
 
-## Specific Problems:
+## Specific Problems
 
 ### SQL queries are not respecting the Reporting Time Zone setting
-#### How to detect this:
+
+**How to detect this:**
 You are not able to click on a cell in a result table or a chart.
 
-#### How to fix this:
+**How to fix this:**
 We do not currently apply a reporting time zone to the results of SQL queries, so you should set one manually.
 
 ### Dates without an explicit time zone are being converted to another day
-#### How to detect this:
-This occurs when you are grouping by a date (vs. a time) that does not have a time zone attached to it. Look at every time field your question uses in the Data Model Reference, and see if any of them are simply a "Date" field.
 
-#### How to fix this:
-You will need to make sure the server time zone reflects the reporting time zone, because when a query is run on Metabase the server applies the time zone to which it is set to that date.
+**How to detect this:**
+This occurs when you are grouping by a date (rather than by a time) that does not have a time zone attached to it. Look at every time field your question uses in the Data Model Reference, and see if any of them are simply a "Date" field.
 
+**How to fix this:**
+Make sure the server time zone reflects the reporting time zone, because when a query is run on Metabase the server applies the time zone to which it is set to that date.
```

Comment:
```suggestion
Make sure the server time zone reflects the reporting time zone, because when a query is run on Metabase, the server applies the configured time zone to that date.
```